### PR TITLE
Fix unnamed macros

### PR DESCRIPTION
**Pull request by vgvassilev**

### CODE REVIEW

**Comment by etejedor** [comment]
Hi @vgvassilev could you resolve the conflicts of this one and merge if it's good to go? Thanks

**Comment by etejedor** [comment]
@vgvassilev what are your plans for this one?

**Comment by lmoneta** [comment]
@vgvassilev  @Axel-Naumann  what do we do with this one ? 
Can the failing tests be fixed ?

File: tutorials/graphics/tmathtext.C [hunk 1]
```diff
@@ -29,5 +29,5 @@
    c1->Print("c1.ps");
    c1->Print("c1.eps");
 
-   return c1;
+   return 0; // success
```
Comment (Axel-Naumann) [inline tutorials/graphics/tmathtext.C]: Returning `c1` means that `c1` will get included in the doc of this tutorial. So this should not be changed.

**Comment by Axel-Naumann** [comment]
projectroot.roottest.root.treeformula.parse.roottest_root_treeformula_parse_make shows an actual error. @vgvassilev I propose we close this for now; you can re-submit once you find the time to address the failure?

**Comment by vgvassilev** [comment]
Sure, I will reopen whenever I am ready.

**Pull request closed without merging**